tests/integration/conftest.py

```diff
@@ -147,6 +154,45 @@ def _create_pidstore(object_uuid, pid_type, pid_value):
     return _create_pidstore
 
 
+@pytest.fixture(scope="function")
+def init_files_db(db):
+    init_default_storage_path()
+    init_records_files_storage_path()
+
+
+@pytest.fixture(scope="function")
+def fsopen_mock():
```

Comment:
shall we move this fixture inside ``integration/records``? It seems related to these test cases only